moz-webgpu-cts/src/main.rs

```diff
@@ -56,22 +56,18 @@ enum Subcommand {
     /// Adjust test expectations in metadata using `wptreport.json` reports from CI runs covering
     /// Firefox's implementation of WebGPU.
     ///
-    /// The general usage of this subcommand is to (1) reset expectations according to some
-    /// heuristic, and then (2) extend expectations from more reports later to accommodate any
-    /// intermittents that are found. More concretely:
+    /// As Firefox's behavior changes, one generally expects CTS test outcomes to change. When you
+    /// are testing your own changes in CI, you can use this subcommand to update expectations
+    /// automatically with the following steps:
     ///
-    /// 1. Pick a `reset-*` preset (which we'll call `RESET_PRESET`). See docs for `preset` for
-    ///    more details on making this choice.
+    /// 1. Run `moz-webgpu-cts process-reports --preset=new-fx …` against the first complete set of
+    ///    reports you gather from CI, to adjust for new permanent outcomes. You may delete the
+    ///    local copies of these reports after being processed.
     ///
-    /// 2. Gather reports into path(s) of your choice.
-    ///
-    /// 3. Run `moz-webgpu-cts process-reports --preset=$RESET_PRESET …` against the reports
-    ///    you've gathered to cover all new permanent outcomes. If you are confident you picked the
-    ///    right `RESET_PRESET`, you may delete the reports you provided to this run.
-    ///
-    /// 4. As intermittent outcomes are discovered (maybe again), run `moz-webgpu-cts
-    ///    process-reports --preset=merge …` with reports. You may delete the reports after their
-    ///    outcomes have been merged in.
+    /// 2. There may still exist intermittent issues that you do not discover in your CI run(s)
+    ///    from the previous step. As you discover them in further CI runs, you can adjust expected
+    ///    outcomes to match with `moz-webgpu-cts process-reports --preset=same-fx …`. You may
+    ///    delete the local copies of these reports after being processed.
```

Comment:
- [ ] nitpick: Coming back a few days after I've written this, I'd like to add to echo some the `new-fx` and `old-fx` languages in the prose, just so they support each other better.